libs/@local/graph/authorization/tests/policies/definitions/mod.rs

```diff
@@ -61,13 +61,16 @@ pub(crate) fn permit_admin_web(web_id: WebId, admin_role: WebRoleId) -> Vec<Poli
 
 pub(crate) fn permit_hash_instance_admins(
     team_id: TeamId,
-    hash_instance_entity_id: EntityUuid,
+    hash_instance_entity_id: EntityId,
 ) -> Vec<Policy> {
     #[expect(clippy::literal_string_with_formatting_args)]
     read_policies(
         &include_str!("permit-hash-instance-admins.cedar")
             .replace("{team_id}", &team_id.to_string())
-            .replace("{entity_id}", &hash_instance_entity_id.to_string()),
+            .replace(
```

Comment:
if this _ever_ becomes a bottleneck (unlikely) (as in - we're replacing 100s of times and therefore allocating a String each time) it may make sense to take a look at https://docs.rs/aho-corasick/latest/aho_corasick/#example-replacing-matches-in-a-stream to do it more efficiently or take a look at https://github.com/askama-rs/askama.

> not important, just putting the ability of aho-corasick to do replacement on our radar that is all^^

Reply:
This is only for seeding some test policies ;)